packages/core/src/indexing/migrations/1-create-model-table.ts

```diff
@@ -32,6 +32,53 @@ function getIndexName(tableName: string): string {
   return tableName.substring(tableName.length - 10)
 }
 
+/**
+ * Default configuration of Compose DB functionality per network.
+ * Values can be overwritten by updating them in the ceramic_config table
+ * and by restarting the node.
+ */
+function getNetworkDefaultConfig(networkName: string): { [key: string]: any } {
+  switch (networkName) {
+    case 'mainnet':
+      return {
+        is_hist_sync: true,
+        allow_queries_before_historical_sync: false,
+        is_run_syncing_worker: true,
```

Comment:
```suggestion
        run_historical_sync_worker: true,
```
maybe? moslty wondering how to make these keys more explicit